frontend/server/src/Controllers/Contest.php

```diff
@@ -2299,9 +2327,16 @@ public static function apiCreate(\OmegaUp\Request $r) {
             'penalty_type' => $r['penalty_type'],
             'languages' => $languages,
             'show_scoreboard_after' => $r['show_scoreboard_after'] ?? true,
+            'contest_for_teams' => $forTeams,
         ]);
 
-        self::createContest($problemset, $contest, $r->user->user_id);
+        self::createContest(
+            $problemset,
+            $contest,
+            $r->user->user_id,
+            /*$originalProblemsetId=*/ null,
+            $forTeams ? $teamsGroupsAlias : null,
```

Comment:
```suggestion
            $teamsGroupsAlias,
```

acá esto ya no sería necesario.

Reply:
Listo